packages/hooks/src/wp-hooks.js

```diff
@@ -1,41 +0,0 @@
-import HOOKS from './hooks';
```

Comment:
The point of this entry file is to build the hooks library for inclusion in core; maybe we can do this directly in core once we add webpack to our build chain. Can you explain how you see this logic being generated with Webpack?